src/jit/hwintrinsicxarch.cpp

```diff
@@ -772,7 +772,7 @@ GenTree* Compiler::impSSE2Intrinsic(NamedIntrinsic        intrinsic,
             CorInfoType corType =
                 strip(info.compCompHnd->getArgType(sig, argList, &argClass)); // type of the second argument
 
-            baseType = JITtype2varType(corType);
+            baseType = JITtype2varTypeExact(corType);
```

Comment:
None of the signatures associated `ConvertScalarToVector128Double` seem to use unsigned types, they only use `Vector<float>`, `Vector<double>`, `int` and `long`. And apparently the name of this variable is wrong, this is not the base type, it's just the type of the second argument. Only later is `baseType` set to the proper type - `TYP_FLOAT`.